acceptance/suites/pre_suite/puppet3_compat/70_install_puppet.rb

```diff
@@ -9,5 +9,13 @@
 
 install_puppet_on(nonmaster_agents, {:version => puppet_version})
 
+variant = master['platform'].variant
```

Comment:
We could also put this in a helper in [helper.rb](https://github.com/puppetlabs/puppetserver/blob/ff0e5e84860ea9b035839e77e9b27d4c3a923e36/acceptance/lib/helper.rb) and have the different pre-suites call into it.  The helper.rb is included in each of the beaker command lines for our different suites.  WDYT?